src/marketing/engagement-cloud.md

```diff
@@ -8,6 +8,7 @@ redirect_from:
 Engagement Cloud, powered by [dotdigital][1], gives you the ability to produce professional, personalized communications and generated reports using data from your Magento store, including:
 
 - Customized email communications
+- [Chat]({% link marketing/engagement-cloud-chat.md -%})
```

Comment:
It's kind of odd that only one of these bullet items is linked

Reply:
I added links to their knowledge base for the rest of the links.